### ISSUE TITLE
LDAP authentication support

### ISSUE THREAD

**Issue by rampatra** [trotto/go-links#44]
Are there any plans to add LDAP authentication support in addition to Google?

Most companies use LDAP for authentication internally so it would be great to have support for the same.

**Comment by itsojon** [comment]
@rampatra yes, there are! To be clear, you're looking to implement authentication for Trotto directly via LDAP (a la https://www.vaultproject.io/docs/auth/ldap), correct?

Also feel free to email us at help@trot.to if you'd like to talk through the details of your requirements.

**Comment by rampatra** [comment]
@itsojon yes, exactly.

Sure but I think you already got it 👍
